include/tvm/runtime/container.h

```diff
@@ -457,6 +470,26 @@ class String : public ObjectRef {
    */
   operator std::string() const { return std::string{get()->data, size()}; }
 
+#if TVM_USE_CXX17_STRING_VIEW
+  /*!
+   * \brief Convert String to an std::string_view object
+   *
+   * \return std::string_view
+   */
+  operator std::string_view() const {
+    return std::string_view{get()->data, size()};
+  }
+#elif TVM_USE_CXX14_STRING_VIEW
+  /*!
+   * \brief Convert String to an std::experimental::string_view object
+   *
+   * \return std::experimental::string_view
+   */
+  operator std::experimental::string_view() const {
+    return std::experimental::string_view{get()->data, size()};
```

Comment:
I am not to sure if we want to convert to experimental string_view, as it is not a part of std yet